cypress/e2e/tests/pages/explorer/api/api-services.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { APIServicesPagePo } from '@/cypress/e2e/po/pages/explorer/api-services.po';
+import SortableTablePo from '@/cypress/e2e/po/components/sortable-table.po';
+
+describe('Cluster Explorer', () => {
+  beforeEach(() => {
+    cy.login();
+  });
+
+  describe('API: APIServices', () => {
+    let apiServicesPage: APIServicesPagePo;
+
+    beforeEach(() => {
+      apiServicesPage = new APIServicesPagePo('local');
+      apiServicesPage.goTo();
+    });
+
+    it('Should have a title', () => {
+      cy.get('h1').should('contain', 'APIServices');
+    });
+
+    it('Should be able to use shift+j to select corre', () => {
+      cy.get('.main-layout').click();
+
+      const sortableTable = new SortableTablePo('.sortable-table');
+      const count = 4;
+
+      cy.keyboardControls({ shiftKey: true, key: 'j' }, 4);
+
+      sortableTable.selectedCountText().should('contain', `${ count } selected`);
+
+      sortableTable.selectedCount().should('eq', count);
```

Comment:
Can the test be expanded to keep hitting the shift + j more times than there are rows? The original [issue](https://github.com/rancher/dashboard/issues/8273) revolved around it continuing to increment past the number of rows